global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "msbuild-sdks": {
+    "Microsoft.NET.Sdk.IL": "3.0.0-preview1-26731-07"
```

Comment:
This should be hooked up to maestro. Not sure how to do that, however.

Reply:
Check out [https://github.com/dotnet/corefx/blob/master/dependencies.props#L181-L185](https://github.com/dotnet/corefx/blob/master/dependencies.props#L181-L185).

1. tools-local\ILAsmVersion.txt should be removed. since it is no longer necessary.
2. That UpdateStep should be changed to instead point to this file, and it should update based on new CoreCLR builds (not BuildTools).
3. You can use a Regex updater for now, but we should probably write an `msbuild-sdks` updater at some point.  /cc @dagood

---
In reply to: [206577961](https://github.com/dotnet/corefx/pull/31512#discussion_r206577961) [](ancestors = 206577961)

Reply:
Along with deleting `tools-local\ILAsmVersion.txt`, we will need to update init-tools to no longer use it.

---
In reply to: [206583480](https://github.com/dotnet/corefx/pull/31512#discussion_r206583480) [](ancestors = 206583480,206577961)

Reply:
We should also update the steps here:

[https://github.com/dotnet/source-build/blob/master/Documentation/boostrap-new-os.md#use-the-bootstrap-cli-to-build-source-build](https://github.com/dotnet/source-build/blob/master/Documentation/boostrap-new-os.md#use-the-bootstrap-cli-to-build-source-build)

> TODO: make this next step better - possibly by adding a new build parameter that takes the path to ILASM. Now, we can initialize our build tools with the bootstrap CLI and the ILASM tool.

---
In reply to: [206584850](https://github.com/dotnet/corefx/pull/31512#discussion_r206584850) [](ancestors = 206584850,206583480,206577961)

Reply:
> You can use a Regex updater for now

I don't think there's currently a way to set up a "pure" regex updater as MSBuild config, unfortunately... we'd need a new case in [BaseDependenciesTask.cs#L95](https://github.com/dotnet/buildtools/blob/ff45aebf7f2f4866558742f8d1b2e3b6e0c52cbe/src/Microsoft.DotNet.Build.Tasks/VersionTools/BaseDependenciesTask.cs#L95) for that. Or just go all the way to a JSON or `msbuild-sdks` updater.

Reply:
Can this be done before merging, or does it need to be done after?

Reply:
Nevermind, it looks like it can all be done as part of this PR.